code/swb_base/util/ScreenUtil.cs

```diff
@@ -65,7 +65,7 @@ public static void ShakeAt(Vector3 origin, float radius = 0, float delay = 0, fl
             rotation *= distanceMul;
             size *= distanceMul;
 
-            ShakeRPC(To.Single(ply), delay, speed, size, rotation);
+            ShakeRPC(To.Single(obj), delay, speed, size, rotation);
```

Comment:
I don't like obj being used as a variable name for a player because it should be clear what data the variable references.

Reply:
Fixed to use the type checked `ModelEntity ply` while waiting for response to the other comment regarding this function.